src/20_Components/amp-date-picker.html

```diff
@@ -137,21 +136,19 @@
   <div>
     <button class="ampstart-btn m1 caps" on="tap:simple-date-picker-3.clear">Clear</button>
```

Comment:
Can you delete the duplicate clear button. The surrounding div is also no longer needed.